src/TransloaditClient.coffee

```diff
@@ -129,21 +118,29 @@ class TransloaditClient
 
     @_remoteJson requestOpts, cb
 
+  streamAssemblies: (params) ->
+    return new PaginationStream (pageno, cb) =>
+      @listAssemblies _.extend({}, params, page: pageno), cb
+
   getAssembly: (assemblyId, cb) ->
     opts =
       url: @_serviceUrl() + "/assemblies/#{assemblyId}"
 
-    @_remoteJson opts, (err, result) =>
-      if err
-        return cb err
+    operation = retry.operation retries: 5, factor: 3.28,
+      minTimeout: 1 * 1000, maxTimeout: 8 * 1000
```

Comment:
Let's use an opts object here or keep the options one on each line for readability please.